### DISCUSSION TITLE
JVM-%EB%B0%91%EB%B0%94%EB%8B%A5-%EA%B9%8C%EC%A7%80-%ED%8C%8C%ED%97%A4%EC%B9%98%EA%B8%B0-ch04-%EA%B0%80%EC%83%81-%EB%A8%B8%EC%8B%A0-%EC%84%B1%EB%8A%A5-%EB%AA%A8%EB%8B%88%ED%84%B0%EB%A7%81%EA%B3%BC-%EB%AC%B8%EC%A0%9C-%ED%95%B4%EA%B2%B0-%EB%8F%84%EA%B5%AC/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#17] [Comments]
# JVM-%EB%B0%91%EB%B0%94%EB%8B%A5-%EA%B9%8C%EC%A7%80-%ED%8C%8C%ED%97%A4%EC%B9%98%EA%B8%B0-ch04-%EA%B0%80%EC%83%81-%EB%A8%B8%EC%8B%A0-%EC%84%B1%EB%8A%A5-%EB%AA%A8%EB%8B%88%ED%84%B0%EB%A7%81%EA%B3%BC-%EB%AC%B8%EC%A0%9C-%ED%95%B4%EA%B2%B0-%EB%8F%84%EA%B5%AC/

양성빈의 블로그

https://sungbin.kr/JVM-%EB%B0%91%EB%B0%94%EB%8B%A5-%EA%B9%8C%EC%A7%80-%ED%8C%8C%ED%97%A4%EC%B9%98%EA%B8%B0-ch04-%EA%B0%80%EC%83%81-%EB%A8%B8%EC%8B%A0-%EC%84%B1%EB%8A%A5-%EB%AA%A8%EB%8B%88%ED%84%B0%EB%A7%81%EA%B3%BC-%EB%AC%B8%EC%A0%9C-%ED%95%B4%EA%B2%B0-%EB%8F%84%EA%B5%AC/

**Comment by lazyshu** [comment]
끼에에에에엑
